src/locales/en-gb.ts

```diff
@@ -153,6 +153,14 @@ const enGB: Locale = {
       close: () => "Close",
     },
   },
+  sort: {
+    accessibleName: (children, sortType) =>
```

Comment:
Suggestion: we could probably go with a more intuitive name here rather than `children`